packages/workbox-window/src/Workbox.ts

```diff
@@ -40,7 +42,25 @@ const REGISTRATION_TIMEOUT_DURATION = 60000;
  *
  * @memberof module:workbox-window
  */
-class Workbox extends EventTargetShim {
+class Workbox extends WorkboxEventTarget {
+  private _scriptURL: 'string';
```

Comment:
What's with the `''` around `'string'`? As far as I can tell, the type is `string`, without quotes: https://www.typescriptlang.org/docs/handbook/basic-types.html#string

Reply:
Good catch! Fixed.